tests/parser/internal/functions.leo

```diff
@@ -0,0 +1,16 @@
+/*
+namespace: ParseInternal
+expectation: Pass
+*/
+
+function a();
+
+function b() -> u32;
+
+function c() {
+	return ();
+}
+
+function d() -> () {
+	return ();
+}
```

Comment:
There's probably a setting in your editor that needs fixing cause of the "no newline at eof"